airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/stream.py

```diff
@@ -213,33 +212,6 @@ def stream_slices(
         for file_info in self.get_time_ordered_file_infos():
             yield {"files": [{"storage_file": self.storagefile_class(file_info, self._provider)}]}
 
-    def _match_target_schema(self, record: Dict[str, Any], target_columns: List) -> Dict[str, Any]:
-        """
-        This method handles missing or additional fields in each record, according to the provided target_columns.
-        All missing fields are added, with a value of None (null)
-        All additional fields are packed into the _ab_additional_properties object column
-        We start off with a check to see if we're already lined up to target in order to avoid unnecessary iterations (useful if many columns)
-
-        :param record: json-like representation of a data row {column:value}
-        :param target_columns: list of column names to mutate this record into (obtained via self._schema.keys() as of now)
-        :return: mutated record with columns lining up to target_columns
-        """
-        compare_columns = [c for c in target_columns if c not in [self.ab_last_mod_col, self.ab_file_name_col]]
-        # check if we're already matching to avoid unnecessary iteration
-        if set(list(record.keys()) + [self.ab_additional_col]) == set(compare_columns):
```

Comment:
`ab_additional_col` removed.
I think yes, it may be a breaking changes, as there might be some users, who are still expect their data in `ab_additional_col`. In this case, They need to make a schema refresh.

Reply:
@clnoll ,
The breaking change is exactly about removing this functionality: removing adding "new" columns in `ab_additional_col` and make it on platform level (by refreshing the schema)

Reply:
Thanks for clarifying @artem1205. Do you know how the platform will handle the changed schema if we're not putting new fields in `ab_additional_col `?